ouroboros-consensus/src/Ouroboros/Consensus/Util.hs

```diff
@@ -444,3 +451,9 @@ instance Bifunctor (StaticEither b) where
 
   second _ (StaticLeft l)  = StaticLeft l
   second f (StaticRight r) = StaticRight (f r)
+
+fromStaticLeft :: StaticEither 'False l r -> l
+fromStaticLeft = \case (StaticLeft x) -> x
+
+fromStaticRight :: StaticEither 'True l r -> r
+fromStaticRight = \case (StaticRight x) -> x
```

Comment:
Is there a particular reason why this written using `LambdaCase`, and not e.g. plainly like this:
```suggestion
fromStaticLeft :: StaticEither 'False l r -> l
fromStaticLeft (StaticLeft x) = x

fromStaticRight :: StaticEither 'True l r -> r
fromStaticRight (StaticRight x) = x
```

Reply:
I copied what I had suggested a while ago, but I don't remember why I chose to use `LambdaCase`. Remove the `LambdaCase`!